django/utils/deprecation.py

```diff
@@ -4,16 +4,16 @@
 from asgiref.sync import iscoroutinefunction, markcoroutinefunction, sync_to_async
 
 
-class RemovedInDjango60Warning(DeprecationWarning):
+class RemovedInDjango61Warning(DeprecationWarning):
```

Comment:
As far as I'm aware, the commit message should be `Advanced deprecation warnings for Django 6.0.` :thinking: 
f39f12030277915f20011f48a0f5eb0d486943b6
b4c5f878bd8d8ba4850ab0f1811cc321e5a627ac
f5060a138583c46e14231b7ca1d256dc1e42f9f9